src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_editor.cljs

```diff
@@ -197,35 +215,42 @@
         [:div {:style {:float "left" :display (when editing? "none") :padding "0.5em 0 1em 0"}}
          (get (config value-type) key)
          (validation-status (contains? invalid-values key))]
-        [:div {:style {:float "left" :display (when-not editing? "none")}}
-         (if (= value-type "inputs")
-           (style/create-text-field {:ref (str "in_" key)
-                                     :defaultValue (get (config value-type) key)})
-           (style/create-text-field {:ref (str "out_" key)
-                                     :defaultValue (get (config value-type) key)}))]
-        (clear-both)
-        [:div {:style {:marginRight "1em" :padding "0.5em" :marginBottom "0.5em"
-                       :backgroundColor (:exception-red style/colors)
-                       :display (if-not (contains? invalid-values key) "none")
-                       :width 450
-                       :border (str "1px solid " (:line-gray style/colors)) :borderRadius 2}}
-         (get invalid-values key)]])]))
+        [:div {}
+         [:div {:style {:float "left" :display (when-not editing? "none")}}
+          (if (= value-type "inputs")
+            (style/create-text-field {:ref (str "in_" key)
+                                      :defaultValue (get (config value-type) key)})
+            (style/create-text-field {:ref (str "out_" key)
+                                      :defaultValue (get (config value-type) key)}))]
+         (clear-both)
+         [:div {:style {:marginRight "1em" :padding "0.5em" :marginBottom "0.5em"
+                        :backgroundColor (:exception-red style/colors)
+                        :display (if-not (contains? invalid-values key) "none")
+                        :width 450
+                        :border (str "1px solid " (:line-gray style/colors)) :borderRadius 2}}
+          (get invalid-values key)]]])]))
+
+
+
```

Comment:
Nitpick - remove a couple of these extra lines. Keep to one or two between methods.